CoreFindView/src/au/gov/asd/tac/constellation/views/find2/components/ReplaceTab.java

```diff
@@ -0,0 +1,178 @@
+ *
+ * @author Atlas139mkm
+ */
+public class ReplaceTab extends BasicFindTab {
+
+    private final Label replaceLabel = new Label("Replace With:");
+    private final TextField replaceTextField = new TextField();
+
+    private final Button replaceNextButton = new Button("Replace Next");
+    private final Button replaceAllButton = new Button("Replace All");
+
+    public ReplaceTab(final FindViewTabs parentComponent) {
+        super(parentComponent);
+        this.setText("Replace");
+        setReplaceGridContent();
+
+        // Sets the actions for the replace all and replace next buttons.
+        replaceAllButton.setOnAction(action -> replaceAllAction());
+        replaceNextButton.setOnAction(action -> replaceNextAction());
+
+    }
+
+    /**
+     * Adds the UI changes for the replace tab. This removes the content from
+     * the unneeded basic find tab and adds anything new specific to the replace
+     * tab
+     */
+    private void setReplaceGridContent() {
+        // Add the replace text box and label
+        textGrid.add(replaceLabel, 0, 1);
+        textGrid.add(replaceTextField, 1, 1);
+
+        // remove the buttons at the bottom of the pane
+        buttonsHBox.getChildren().clear();
+
+        // add the replace buttons
+        buttonsHBox.getChildren().addAll(replaceNextButton, replaceAllButton);
+
+        // remove addTo, findIn, removeFrom
+        currentSelectionChoiceBox.getItems().remove(1, 4);
+
+        // add replaceIn
+        currentSelectionChoiceBox.getItems().add("Replace in");
+
+        // remove exact match checkBox
+        preferencesGrid.getChildren().remove(exactMatchCB);
+    }
+
+    /**
+     * Updates the buttons at the bottom of the pane to be specific to the
+     * replace tab.
+     */
+    public void updateButtons() {
+        buttonsHBox.getChildren().clear();
+        buttonsHBox.getChildren().addAll(replaceNextButton, replaceAllButton, searchAllGraphs);
+        getParentComponent().getParentComponent().setBottom(buttonsVBox);
+    }
+
+    /**
+     * Reads the find text, element type, attributes selected, standard text
+     * selection, regEx selection, ignore case selection, exact match selected
+     * and search all graphs selection and passes them to the controller to
+     * create a BasicFindReplaceParameter
+     */
+    public void updateBasicReplaceParamters() {
+        // get the currently selected graphElementType
+        final GraphElementType elementType = GraphElementType.getValue(lookForChoiceBox.getSelectionModel().getSelectedItem());
+
+        // get the matching attributeList
+        final List<Attribute> attributeList = new ArrayList<>(getMatchingAttributeList(elementType));
+        boolean replaceIn = false;
+
+        // determine what currentSelectionChoiceBox option is selected
+        switch (currentSelectionChoiceBox.getSelectionModel().getSelectedIndex()) {
+            case 0:
+                break;
```

Comment:
given this is the same as the default case, I would either omit this altogether or shift it below `case 1` if you do want to explicilty mention it. i.e.:
```
...
case 0:
default:
  break;
```